packages/wonder-blocks-form/src/components/text-field.stories.js

```diff
@@ -40,6 +40,30 @@ export const Text: StoryComponentType = () => {
     );
 };
 
+export const Required: StoryComponentType = () => {
+    const [value, setValue] = React.useState("");
+
+    const handleChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    const handleKeyDown = (event: SyntheticKeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
+
+    return (
+        <TextField
+            id="tf-1"
+            type="text"
+            onChange={handleChange}
+            onKeyDown={handleKeyDown}
+            required={true}
+        />
```

Comment:
oops, I forgot to add `value={value}` here.

Why do we need to do it that way anyway? When I use `value=""`, it doesn't let me type anything into the box